src/browser/index.jsx

```diff
@@ -508,13 +508,9 @@ var showUnreadBadgeOSX = function(unreadCount, mentionCount) {
 }
 
 var showUnreadBadgeLinux = function(unreadCount, mentionCount) {
-  /*if (mentionCount > 0) {
-    remote.app.dock.setBadge(mentionCount.toString());
-  } else if (unreadCount > 0) {
-    remote.app.dock.setBadge('•');
-  } else {
-    remote.app.dock.setBadge('');
-  }*/
+  if (remote.app.isUnityRunning()) {
```

Comment:
is this check really necessary? looks like it prevents the counter from working in Ubuntu GNOME...

Reply:
The method name is misleading. isUnityRunning checks if libunity is available which is the case for the default Gnome distribution on Ubuntu for Desktop.
Please see the related discussion here: https://github.com/electron/electron/issues/18349

Reply:
@jnugh thanks for explaining. Then the question remains why I don't get a badge on Ubuntu 19.04 (Mattermost installed via snap). Any clue?